bgpd/bgp_vty.c

```diff
@@ -10499,6 +10499,25 @@ static void bgp_show_failed_summary(struct vty *vty, struct bgp *bgp,
 	}
 }
 
+/* If the peer's description includes whitespaces
+ * then return the first occurrence. Also strip description
+ * to the given size if needed.
+ */
+static char *bgp_peer_description_stripped(char *desc, uint32_t size)
+{
+	static char stripped[BUFSIZ];
+	char *pnt;
```

Comment:
define a variable
```
#define  DESC_STRIP_LEN 21
```
and use it in bgp_vty.c too

Reply:
What if I want to use a wider form of description? Like `show bgp summary wide` and print a larger description? (command does not exists at the moment, but it's in my head to add)

Reply:
ah ok,
because the param you passed stands for the size.
ok, then lets cancel my comment. it is ok like that.